From 93410fdf81df9b4dddaaf57ebab815fcd4ded56f Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Thu, 4 Jan 2024 13:58:29 +0100
Subject: [PATCH 0362/1695] dts: stm32: enable RTC on stm32mp257f-dk

Set RTC status to "okay" in stm32mp257f-dk.dts

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git e2daa7066070a0b7d12882d8dcdba300ac068b32 dts: stm32: enable RTC on stm32mp257f-dk]
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Change-Id: I6b22f0184e95679e61df21534edcbdeec924c858
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/349408
Domain-Review: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/arch/arm/dts/stm32mp257f-dk.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp257f-dk.dts b/core/arch/arm/dts/stm32mp257f-dk.dts
index bbf55079b..0233ca5b9 100644
--- a/core/arch/arm/dts/stm32mp257f-dk.dts
+++ b/core/arch/arm/dts/stm32mp257f-dk.dts
@@ -46,6 +46,10 @@
 	status = "okay";
 };
 
+&rtc {
+	status = "okay";
+};
+
 &usart2 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&usart2_pins_a>;
-- 
2.47.2

